nncf/common/accuracy_aware_training/training_loop.py

```diff
@@ -209,6 +209,11 @@ def __init__(self,
             raise RuntimeError('No compression algorithm supported by the accuracy-aware training '
                                'runner was specified in the config')
 
+        if maximal_compression_rate > self.adaptive_controller.maximal_compression_rate:
+            maximal_compression_rate = self.adaptive_controller.maximal_compression_rate
+            nncf_logger.warning(f'Updated training loop maximal compression rate by the maximal compression rate '
```

Comment:
Probably, the maximum compression rate in 95% can not be archived for most models. I mean that the user will see warning if maximal_compression_rate will not defined and use default value 0.95. It can be confuse an user. I would recommend to handle it during the accuracy aware loop.

Reply:
Agree. Let's keep only the logging during the loop itself.